shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/openapi/OpenApiReaderImplementation.java

```diff
@@ -22,15 +22,17 @@ public static OpenApiReaderImplementation getInstance() {
      * Loads the stream of an OpenApi file and returns the output as a string.
      *
      * @param fileName Name of the file to load
+     * @param charset Charset used for the output string
      * @return File contents as string
      * @throws UnableToReadOpenApiSpecificationException If there is an issue loading the Api file
      */
     @Override
-    public String openAsString(String fileName) throws UnableToReadOpenApiSpecificationException {
-        InputStream stream = getClass().getClassLoader().getResourceAsStream(fileName);
-        try {
-            return new String(
-                    Objects.requireNonNull(stream).readAllBytes(), StandardCharsets.UTF_8);
+    public String openAsString(String fileName, Charset charset)
+            throws UnableToReadOpenApiSpecificationException {
+        try (InputStream stream = getClass().getClassLoader().getResourceAsStream(fileName)) {
+            var out = new String(Objects.requireNonNull(stream).readAllBytes(), charset);
```

Comment:
I don't think we need the `Objects.requireNonNull()`.  `stream` will be not be `null` if we get to this point.

Reply:
I have that there to silence a warning on the IDE, but since we're catching the NullPointerException that it's worried about it's probably a false positive. Will do!